app/components/layout/Navigation/index.js

```diff
@@ -21,7 +21,8 @@ class Navigation extends React.Component {
   static propTypes = {
     organization: React.PropTypes.object,
     viewer: React.PropTypes.object,
-    relay: React.PropTypes.object.isRequired
+    relay: React.PropTypes.object.isRequired,
+    assumed: React.PropTypes.bool
```

Comment:
I'd probably do something super basic like: `window._navigation = { warning: true }`. Check for that instead of `this.props.assumed`, and that'd be super simple to implement in the backend.